Desktop/Prework/Allegro/allegro/src/components/ListItem/ListItem.module.scss

```diff
@@ -0,0 +1,76 @@
+.wrapper{
+    width: 11rem;
+    height: 17rem;
+    position: relative;
+    background: #FFFFFF;
+    border: 2px solid rgba(0, 0, 0, 0.9);
```

Comment:
A very good practice is to have all the colors defined in one file as either scss or css variables. This way you have control over how many colors are there in the app. The same thing is with fontSizes and spaceSizes